### PR TITLE
redirect and redeem outstanding dropt3 rewards to treasury ops

### PR DESCRIPTION
**Pull request by jalbrekt85**
**solves #217**:  add script to claim tree reward from trops and redeem for bdigg and withdraw to digg(?)
run:
`brownie run issue/217/claim_dropt.py`

### CODE REVIEW

File: scripts/issue/217/claim_dropt.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    safe = GreatApeSafe(registry.eth.badger_wallets.treasury_ops_multisig)
+    safe.init_badger()
+
+    digg = registry.eth.treasury_tokens.DIGG
+    bdigg = interface.ISettV4h(registry.eth.treasury_tokens.bDIGG, owner=safe.address)
+    dropt = safe.contract(registry.eth.uma.DIGG_LongShortPair)
+    dropt_long = safe.contract(dropt.longToken())
+
+
+    safe.take_snapshot(tokens=[
```
Comment (petrovska-petro) [inline scripts/issue/217/claim_dropt.py:20]: scripts looks clean, but will be great to add also in the snap `bcvxCRV`and `bveCVX`as they are claim also from the tree

Comment (jorijnsmit) [reply scripts/issue/217/claim_dropt.py:20]: +1

Comment (jalbrekt85) [reply scripts/issue/217/claim_dropt.py:20]: good idea 👍

**Comment by jorijnsmit** [comment]
as mentioned by @petrovska-petro, still some dropt3 in the tree remains. @Darruma is working on a fix for this if i am not mistaken. lets wait for that before merging.
```
snapshot result for 0x660802Fc641b154aBA66a62137e71f331B6d787A:
                     balance_before               balance_after               balance_delta
symbol                                                                                     
DROPT-3   34,380.917097829000000000    1,247.015660680000000000  -33,133.901437149000000000
bcvxCRV  146,411.523652158403241093  145,579.618274305256354472     -831.905377853146886621
bveCVX    65,811.304319315145732649   65,730.218102140793344036      -81.086217174352388613
```
```
snapshot result for 0x042B32Ac6b453485e357938bdC38e0340d4b9276:
                    balance_before              balance_after           balance_delta
symbol                                                                               
DIGG          4.425255369000000000      10.014513016000000000    5.589257647000000000
bcvxCRV     481.819857877370283228   1,313.725235730517169849  831.905377853146886621
bveCVX   23,509.210589095968722220  23,590.296806270321110833   81.086217174352388613
```

**Review by petrovska-petro — APPROVED** [review]

**Review by jorijnsmit — APPROVED** [review]
beautiful, tree issue has been fixed.

**Pull request merged**